Documentation/user-guides/getting-started.md

```diff
@@ -274,50 +107,49 @@ spec:
   enableAdminAPI: false
```

Comment:
Not sure if these parts make sense in this guide 🤔

Reply:
Agreed

Reply:
I suggest that the first step of the guide shows the Prometheus resource with all selectors set. IMHO it is easier to get than going back and forth.